### DISCUSSION TITLE
Why should I use Havit over just using Boostrap directly?

### DISCUSSION THREAD

**Discussion by oscar230** [havit/Havit.Blazor#240] [General]
I know this is a simple question, but that's my intention, I want to keep it open ended and hear how people use this library.
It looks really cool and I want to use it, although I did not see any real use cases or anecdotes on the website.

Sincerely, Oscar.

**Comment by hakenr** [comment]
We (in HAVIT) use the library (not just the UI components, but the whole stack incl. gRPC communication etc., see https://github.com/havit/NewProjectTemplate-Blazor) for almost all our new projects (started in 2021 and later). We do custom development, mainly line-of-business apps, e-commerce apps, intranet apps. The library brings us (over BS):
* ease of use (IntelliSense, Blazor lifecycle, ...)
* extra components not covered in plain Bootstrap (HxGrid, HxSidebar, HxTreeView, HxCalendar, HxInputDate, HxAutosuggest, HxSearchBox, ...)
* shared knowledge for the team ("this is how you should do it" :-))

Beside the NewProjectTemplate-Blazor, there are a few public projects where you can see the full stack in action:
* https://github.com/havit/Bonusario - merit-money "voting" system,
* https://github.com/mensagymnazium/IntranetGen3 - school intranet (choosing electives, etc.)

**Reply by oscar230** [reply]
Thank you for making this @hakenr, I got recommended to check this out from my senior developer/mentor. I am really impressed, been playing around with it a bit now! Great work, and thanks for the explanation, I can really see why its not "just boostrap 5 in C#", it's way more than that! Cool. 🚀
